app/Presenters/AssetPresenter.php

```diff
@@ -233,18 +233,28 @@ public static function dataTableLayout()
                 'title' => trans('general.user_requests_count'),
 
             ], [
-                'field' => 'created_at',
```

Comment:
Does this mean that we no longer have `created_at` in the AssetPresenter?

Reply:
No, it was just moved - git is making it look confusing. 


https://github.com/user-attachments/assets/9f5e857e-e9b2-431b-875d-b125b27d66d1